### ISSUE TITLE
No way to escape placeholders in forwardSearch.args

### ISSUE THREAD

**Issue by eburghar** [latex-lsp/texlab#649]
[okular](https://docs.kde.org/stable5/en/okular/okular/configeditor.html) also use `%f`, `%l` for the inverse search command, and it has the possibility to configure the editor from the command line with the `--editor-cmd` argument.

But this is not working

```json
{
  "texlab.forwardSearch.executable": "okular",
  "texlab.forwardSearch.args": ["--editor-cmd": "\"code -g \"%f\":%l\"", "--unique", "file:%p#src:%l%f"]
}
```

We can enclose an argument with " to prevent placeholders replacement but as the leading and trailing " are not removed from the argument, okular fails to launch the command.

A `%` would be preferable to escape `%`. I have a PR that  permits this. Tell me if this makes sense.

```json
{
  "texlab.forwardSearch.executable": "okular",
  "texlab.forwardSearch.args": ["--editor-cmd": "code -g \"%%f\":%%l", "--unique", "file:%p#src:%l%f"]
}
```

**Comment by pfoerster** [comment]
Thanks for the report. 

> A `%` would be preferable to escape `%`.

I agree. The current replacing strategy cannot really handle this case so an escape character still lets the old configurations work while fixing this issue.

> I have a PR that permits this. Tell me if this makes sense.

Sure! Thanks for contributing 👍

**Comment by pfoerster** [comment]
Released with `texlab 4.2.0`
